development/submissions.html

```diff
@@ -264,39 +264,38 @@
               managing a review is a serious commitment; for this reason,
               it's better to contact the member off-list.</p>
 
-              <p>Once a potential review manager has been found, <a
+              <p>Once a potential review manager has been identified, <a
               href="/community/reviews.html#Wizard">contact the
               review wizards</a> for approval. The wizards approve review
               managers based on their level of participation in the Boost
               community.</p>
 
-              <p>The review wizards will coordinate with the author and
-              review manager to schedule a date convenient for the author and
-              review manager.</p>
+              <p>The review wizards will coordinate with both the author and
+              review manager to schedule a date convenient for both.</p>
 
               <p>See <a href="/community/reviews.html">Formal Review
               Process</a> for details.</p>
 
 
               <h2><a name="Review" id="Review"></a>7. Formal Review</h2>
 
-              <p>Before your formal review begins, quadruple check your
-              library. Check that every code example works, that all unit
-              tests pass on at least two compilers on at least two major
-              operating systems, and run your documentation through a
-              spelling and grammar checker.</p>
+              <p>Before your formal review begins, double-, triple-, and
+              quadruple-check your library. Verify that every code example
+              works, that all unit tests pass on at least two compilers on at
+              least two major operating systems, and run your documentation
+              through a spelling and grammar checker.</p>
 
               <p>Please do not modify your library on its master branch
-              during a review. Instead modify develop branch in response
-              to feedback and reviews. For bigger ticket items of work,
-              open issues on your issue tracker so interested people can
+              during a review. Instead, modify a separate develop branch in
+              response to feedback and reviews. For bigger ticket items of
+              work, open issues on your issue tracker so interested people can
               track the fixing of specific issues raised.</p>
 
               <p>The review manager will consider all the reviews made by
               members of the community and arrive at a decision on
               whether your library is rejected, conditionally accepted or
-              unconditionally accepted. They will post a report summarising
-              their decision publicly. If conditions are attached to
+              unconditionally accepted. He or she will post a report summarising
```

Comment:
What about the people who don't identify as plural? Feel free to do whatever you think is best here.